### ISSUE TITLE
Fix: video recording could be supported in iOS 16

### ISSUE THREAD

**Issue by samurm** [Cap-go/capacitor-camera-preview#140]
Hi, really good plugin.

I'm doing a fork with little changes that I need for supporting depth params.

And I'm testing in my Iphone X with iOS 16.7.10, at the beginning I saw this:

```
if #unavailable(iOS 17) {
    call.reject("You cannot record video on IOS < 17")
}
```

But when I removed that condition and I tried to record a video, it works really good on my device.
I tested record, flip, stop and reproduce in a video html element.

Are there any reason to only available iOS 17?
I didn't see anything in the documentation. 

Thank you!

**Comment by riderx** [comment]
Thanks for the feedback, I think @WcaleNieWolny has the answer.

**Comment by riderx** [comment]
Can you make a PR with your changes? we would love to have them

**Comment by WcaleNieWolny** [comment]
@riderx I’m not certain what led you to assume I have the necessary information. I am, at the time of writing this, not in possession of an old iPhone with iOS 16 and as such I am unable to assist with the resolution of this issue.

**Comment by riderx** [comment]
I have asked you because the limit was added by you on git blame ^^

**Comment by WcaleNieWolny** [comment]
I frankly don't remember ^^

I do think that I might have implemented video recording, as it wasn't not implemented when you asked me to look at this plugin, but I might be mistaken on this.

**Comment by riderx** [comment]
Ok let’s remove it then

**Comment by riderx** [comment]
removed and pushed

**Comment by riderx** [comment]
released in [6.5.8](https://github.com/Cap-go/camera-preview/tree/refs/tags/6.5.8)
